src/utils/output.ts

```diff
@@ -80,6 +85,19 @@ export function displaySuccesses(result: DeployResult | RetrieveResult): void {
   cli.table(successes, columns, options);
 }
 
+export function displayPackages(result: RetrieveResult, packages: PackageRetrieval[]): void {
```

Comment:
function to display packages in their own output table

Reply:
Nice touch.